pkg/handlers/notes.go

```diff
@@ -74,7 +77,43 @@ func (h NotesHandler) Handle() http.HandlerFunc {
 			}
 
 		case "POST":
-			h.WriteErrorResponse(r.Context(), w, errors.New("not yet implemented"))
+			if r.Body == nil {
+				h.WriteErrorResponse(
+					r.Context(),
+					w,
+					&apperrors.BadRequestError{Err: errors.New("empty request not allowed")},
+				)
+				return
+			}
+			defer r.Body.Close()
+
+			decoder := json.NewDecoder(r.Body)
+			note := models.Note{}
+			note.SystemIntakeID = uuid
```

Comment:
I think you may want to do this _after_ the decode... It is possible that the JSON body overwrites this value.

Reply:
Also, I would suggest using the Principal to overwrite the EUAID, either here on maybe in the services layer. (This is an artifact of how we have data and metadata co-habitating on the single type...)

Reply:
I think I did this and didn't commit/push the code. But agreed.

Reply:
Ok. I am grabbing the id of the user from the Principal in the service. But I will move the write of the uuid from the route to happen after the decode. Thanks for pointing that out!